cli/endpoints/batch/deploy-pipelines/batch-scoring-with-preprocessing/components/prepare/prepare.yml

```diff
@@ -0,0 +1,43 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
+name: uci_heart_prepare
+display_name: UCI Heart dataset prepare
+type: command
+description:  |-
+  **Prepare UCI Heart dataset**:
+
+  Prepares the UCI Heart datasets and outputs the transformation's parameters learnt during the preprocessing. If the transofrmations are also provided as inputs, then no parameters are learnt and the same transformations indicated are applied to the data.
+
+  The following parameters can be indicated:
+
+  * `data`: the data to transform.
+  * `transformations`: the input transformations to apply to the data for scaling the data. If no provided, then the transformations are learnt.
+  * `categorical_encoding`: the categorical encoding to use if transformations are learnt. Possible values are `onehot` or `ordinal`.
+
+  The following outputs are produced:
+
+  * `prepared_data`: the prepared data.
+  * `transformations_output`: the transformations applied to the input data. If the transformation were provided as inputs, then this output has the same values that the input `transformation`.
```

Comment:
change `that` to `as`

Reply:
values that the input `transformation`. -> values as the input `transformation`.

Reply:
Done!